stablehlo/tests/vhlo/vhlo_to_version_downgrade_invalid.1_1_0.mlir

```diff
@@ -13,3 +13,15 @@ func.func @type_ui2(%arg0: tensor<ui2>) -> tensor<ui2> {
   %0 = stablehlo.add %arg0, %arg0 : tensor<ui2>
   func.return %0 : tensor<ui2>
 }
+
+// -----
+
+func.func @custom_call_dictionary_attr(%arg0: tensor<f32>) -> tensor<f32> {
```

Comment:
Testcase looks good, but after a version bump it should go in a new file: `stablehlo/tests/vhlo/vhlo_to_version_downgrade_invalid.1_2_0.mlir`, and we need to bump the minor version again to 1.3.0 since there is an opset change in this PR (will leave separate comment to track this).

We should also add a test for default valued dict (leave off backend config, but specify v4)

Reply:
> We should also add a test for default valued dict (leave off backend config, but specify v4)

Moved this feedback to a new comment.

Reply:
1. moved tests to  a new file. 
2. added similar test but without backend_config